.cargo/config.toml

```diff
@@ -1,5 +1,6 @@
 [alias]
 xtask = "run --package xtask --quiet --"
+phd = "run --package cargo-phd --quiet --"
```

Comment:
This seems like a really cool use of aliases! Not a request, just a question, but why bother using xtask when all subcommands could be implemented this way?

Reply:
Speaking personally, I like the namespacing offered by an xtask prefix.  `cargo xtask clippy`, for example, performs the run(s) with all the necessary fanfare, and I wouldn't want it colliding with the native `cargo clippy`

Reply:
I'm happy to change this to `cargo xtask phd` if that's preferred! My initial approach was going to use Cargo artifact deps, so I didn't want it to be part of the `xtasks` crate so that compiling xtasks didn't also require compiling `propolis-server` and `phd-runner`. But, I ended up not using artifact dependencies, since they're unstable, and instead did the compilation of `propolis-server` and `phd-runner` at runtime, so that issue is no longer the case. I'm happy to move this code to the `xtasks` crate if you like!

Reply:
I'm not opposed to it living outside of `xtask`.  I was more voicing my opposition to all xtasks being turned into aliases.

Reply:
I moved it into the `xtasks` crate in 0da065452a5283bf79259976bfcbaa6b17b97325 --- it is nice for it to show up in `cargo xtask help`.

Reply:
Sounds good.